modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/KeepWordFilterFactory.java

```diff
@@ -54,6 +54,7 @@
     private final CharArraySet keepWords;
     private static final String KEEP_WORDS_KEY = "keep_words";
     private static final String KEEP_WORDS_PATH_KEY = KEEP_WORDS_KEY + "_path";
+    @SuppressWarnings("unused")
     private static final String KEEP_WORDS_CASE_KEY = KEEP_WORDS_KEY + "_case"; // for javadoc
```

Comment:
The "_case" Setting is used in Analysis#getWordSet, but since that method also serves e.g. WordDelimiterTokenFilterFactory, it cannot use this concatenated constant. I think it still makes sense to keep it here for the docs though.